src/main/java/edu/cornell/kfs/coa/batch/businessobject/LaborClosedAccount.java

```diff
@@ -0,0 +1,85 @@
+package edu.cornell.kfs.coa.batch.businessobject;
+
+import java.sql.Date;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+
+public class LaborClosedAccount {
+    
+    private String chart;
+    private String accountNumber;
+    private String subAccountNumber;
+    private String accountClosedIndicator;
+    private Date accountClosedDate;
+    
+    public LaborClosedAccount() {
+        this.chart = null;
+        this.accountNumber = null;
+        this.subAccountNumber = null;
+        this.accountClosedIndicator = null;
+        this.accountClosedDate = null;
+    }
+    
+    public LaborClosedAccount(String chart, String  accountNumber, String subAccountNumber, String accountClosedIndicator, Date accountClosedDate) {
+        this.chart = chart;
+        this.accountNumber = accountNumber;
+        this.subAccountNumber = subAccountNumber;
+        this.accountClosedIndicator = accountClosedIndicator;
+        this.accountClosedDate = accountClosedDate;
+    }
+    
+    public String getChart() {
+        return chart;
+    }
+    
+    public void setChart(String chart) {
+        this.chart = chart;
+    }
+    
+    public String getAccountNumber() {
+        return accountNumber;
+    }
+    
+    public void setAccountNumber(String accountNumber) {
+        this.accountNumber = accountNumber;
+    }
+    
+    public String getSubAccountNumber() {
+        return subAccountNumber;
+    }
+    
+    public void setSubAccountNumber(String subAccountNumber) {
+        this.subAccountNumber = subAccountNumber;
+    }
+    
+    public String getAccountClosedIndicator() {
+        return accountClosedIndicator;
+    }
+    
+    public void setAccountClosedIndicator(String accountClosedIndicator) {
+        this.accountClosedIndicator = accountClosedIndicator;
+    }
+
+    public Date getAccountClosedDate() {
+        return accountClosedDate;
+    }
+
+    public void setAccountClosedDate(Date accountClosedDate) {
+        this.accountClosedDate = accountClosedDate;
+    }
+    
+    public String toCsvString() {
+        SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy", Locale.US);
```

Comment:
"MM/DD/yyyy" is defined as a constant in KFSConstants

Reply:
Changed.